upgrade_logic/business_objects/gateway.py

```diff
@@ -1,8 +1,47 @@
 import re
-from submodules.model.business_objects import attribute, embedding, user, general
+from config_handler import get_config_value
+from submodules.model.business_objects import (
+    attribute,
+    embedding,
+    user,
+    general,
+    organization,
+)
 from submodules.model import enums
 
 
+def gateway_1_8_1() -> bool:
+    __gateway_1_8_1_add_organization_limits()
+    return True
+
+
+def __gateway_1_8_1_add_organization_limits() -> bool:
+    is_managed = get_config_value("is_managed")
+    if is_managed:
+        max_rows = 50000
+        max_cols = 25
+        max_char_count = 100000
+    else:
+        max_rows = get_config_value("max_rows") or 50000
+        max_cols = get_config_value("max_cols") or 25
+        max_char_count = get_config_value("max_char_count") or 100000
+
+        print(
+            f"Add default limit for organizations",
+            flush=True,
+        )
+        general.execute(
```

Comment:
i think these have the wrong indentation.

Reply:
done